tests/testthat/test-lines.R

```diff
@@ -16,3 +18,53 @@ test_that("lines don't have fill", {
   })
   expect_equal(xml_attr(xml_find_one(x, ".//polyline"), "fill"), "none")
 })
+
+
+test_that("lines lty becomes stroke-dasharray", {
+  stroke_dash <- c( "5,5", "1,5", "1,5,5,5", "10,5", "10,5,5,5" )
+
+  # base graphics
+  x <- xmlSVG( plot( 1:3, lty = 1, type = "l" ) )
+  expect_equal(
+    xml_attr(suppressWarnings(xml_find_one(x,"//polyline")), "stroke-dasharray"),
+    NA_character_
+  )
+
+  expect_identical(
+    sapply(
```

Comment:
This formulation puts the expected and actual values very far apart. I'd write it like:

```R
dash_array <- function(lty) {
   x <- xmlSVG(plot(1:3, lty = lty, type = "l"))
   xml_attr(xml_find_one(x, "//polyline"), "stroke-dasharray")
}
expect_equal(dash_array(1), "5,5")
```

etc